GameData/ROEngines/PartConfigs/GEM40_BDB.cfg

```diff
@@ -0,0 +1,102 @@
+PART
+{
+	module = Part
+	name = ROE-GEM40
+	author = Cobaltwolf, Vader111
+
+	RSSROConfig = true
+	ROESetEngineDefaults = SOLID_BOOSTER
+
+	mass = 1.1879
+
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Delta/bluedog_Delta_GEM40
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_attach = 0.0, 0.0, -0.3125, 0.0, 0.0, 1.0, 1
+	CoMOffset = 0.0, -0.16, -0.16
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0	
+
+	title = GEM 40
+	manufacturer = Hercules
+	description = The Graphite-Epoxy Motor (GEM) replaced the steel case used on earlier Castor-series boosters with a lighter composite case. The 40-inch (1-meter) diameter GEM 40 was used on the Delta II 7000-series in sets of three, four, or nine. When nine boosters were used, six were ignited at liftoff and the remaining three were ignited after burnout and jettison of the first six. Burn time: 58 seconds.
+
+	tags = ATK, GEM40, booster, radial, solid, srb,
+
+	engineType = GEM-40
+
+	skinInternalConductionMult = 4.0
+	emissiveConstant = 0.5
+	dragModelType = default
+	maximum_drag = 0.3
+	minimum_drag = 0.2
+	angularDrag = 2
+
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchNose
+		switcherDescription = Nose Shape
+		switcherDescriptionPlural = Nose Shapes
+		affectDragCubes = True
+		affectFARVoxels = True
+		SUBTYPE
+		{
+			name = Straight
+			transform = StraightNose
+		}
+		SUBTYPE
+		{
+			name = Angled
+			transform = AngledNose
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleSurfaceFX
+		thrustProviderModuleIndex = 0
+		fxMax = 1
+		maxDistance = 80
+		falloff = 2
+		thrustTransformName = thrustTransform
+	}
+	
+	MODULE
+	{
+		name = FXModuleAnimateThrottle
+		animationName = GEM40_Emit_Anim
+		dependOnEngineState = True
+		responseSpeed = 0.001
+	}
+
+}
+
+@PART[ROE-GEM40]:AFTER[RealismOverhaulEnginesPost] 
+{
+	@MODULE[ModuleEngineConfigs] 
+	{
+		@CONFIG[GEM-40/Ground]
+		{
+			@origMass = 1.1879
+		}
+		@CONFIG[GEM-40/Air]
+		{
+			@origMass = 1.3269
+		}
+	}
```

Comment:
If the mass values in the RO engine config are incorrect, please PR the fix there and I will merge it, rather than patching over top.  You can adjust the origMass (overall) or the massMult (per-config).  (ROEngines is not going to disagree with RO core about the mass of a part.)